### DISCUSSION TITLE
❗ Add the possibility to add Help Text to checkbox

### DISCUSSION THREAD

**Discussion by alexasselin008** [gsoft-inc/sg-orbit#1124] [Ideas]
## Description

Checkbox and Radio often have a description or help text. 

![image](https://user-images.githubusercontent.com/38871812/194615016-4f22d8a5-d614-4db9-bd11-e4fdf16b29af.png)

![image](https://user-images.githubusercontent.com/38871812/194614929-333c99cd-6978-4322-903b-74d0268c8e04.png)
![image](https://user-images.githubusercontent.com/38871812/194615078-634fe2fa-d5d1-4e14-924a-29c9de821b5d.png)
![image](https://user-images.githubusercontent.com/38871812/194615163-b6254f77-6743-45ef-935c-fd313ca1768c.png)

Inspiration: 
https://polaris.shopify.com/components/radio-button

**Comment by patricklafrance** [comment]
I suggest moving this to a Github discussion.
